File: code/R/R Computational Toolbox Files/SocialNetworks R/degPersonPPG.R
# Callixte Nahimana
# Wofford College, Spartanburg, SC
# April, 18 2014

degPersonPPG<-function(adjPeopleMat, i ){
#DEGPERSONPPG returns the degree of a person node in people-to-people graph
#   Given a people-to-people matrix (adjPeopleMat) and an index i, 
#   the function returns the degree of that node.

return(sum(adjPeopleMat[i,]));
}

File: products/Introduction_to_Computational_Science_Modules/02_System_Dynamics/MATLAB/Files/SocialNetworksMATLAB/degPersonPPG.m
function [ deg ] = degPersonPPG(adjPeopleMat, i )
%DEGPERSONPPG returns the degree of a person node in people-to-people graph
%   Given a people-to-people matrix (adjPeopleMat) and an index i, 
%   the function returns the degree of that node.

deg = sum(adjPeopleMat(i,:));

end

